### ISSUE TITLE
Mediapipe Hands - Hand Geomtery

### ISSUE THREAD

**Issue by rkgnanesh** [google/mediapipe#4621]
### MediaPipe Solution (you are using)

Hands

### Are you willing to contribute it

None

### Describe the feature and the current behaviour/state

```shell
We need hand geometry in mediapipe hands solution, just as we have in facemesh face geometry
```

### Please specify the use cases for this feature

```shell
The placement of 3D objects on hand can be made more accurate
```

**Comment by jashwanth9845** [comment]
Yeah, that would be more useful if mediapipe updated this feature to work with the Mediapipe hands library

**Comment by kuaashish** [comment]
@jashwanth9845,

Thank you for your thoughts on extending the functionality of hands. We are forwarding request internally and based on discussion, Will prioritise the feature.

**Comment by git1bhay** [comment]
yes hand geometry we want.
